Document common ActiveStorage issues (#48722)
rails/rails: guides/source/active_storage_overview.md

If you don’t provide a content type and Active Storage can’t determine the
file’s content type automatically, it defaults to application/octet-stream.

> ### Replace vs Add Attachments

Reviewer: ```suggestion
### Replacing vs Adding Attachments
```

By default in Rails, attaching files to a `has_many_attached` association will replace
any existing attachments. To instead add new attachments while retaining existing
ones, set `Rails.application.config.replace_on_assign_to_many` to false.

> Alternatively, you can add hidden form fields with the [`signed_id`][ActiveStorage::Blob#signed_id]
> of each attached file:

Reviewer: We try to avoid using "you" in wording for Rails docs: https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#wording 😄 

```suggestion
Alternatively, use hidden form fields with the [`signed_id`][ActiveStorage::Blob#signed_id]
of each attached file:
```
